mail_improved_tracking_value/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Improved tracking value change',
+    'version': '10.0.1.0.0',
+    'summary': 'Improves tracking changes by adding many2many and one2many \
+                    support. As well as a view to consult changes',
```

Comment:
Additionally there is a bunch of whitespace being added here. The strings should instead be started and stopped.

'my first line. '
'my second line.'